pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -753,8 +755,8 @@ def handle_lazy_main_feasible_solution_gurobi(cb_m, cb_opt, solve_data, config):
             cb_opt.cbGet(gurobipy.GRB.Callback.MIPSOL_OBJBND), solve_data.UB)
         solve_data.bound_improved = solve_data.UB < solve_data.UB_progress[-1]
         solve_data.UB_progress.append(solve_data.UB)
-    config.logger.info(
-        'MIP %s: OBJ (at current node): %s  Bound: %s  LB: %s  UB: %s  TIME: %s'
-        % (solve_data.mip_iter, cb_opt.cbGet(gurobipy.GRB.Callback.MIPSOL_OBJ), cb_opt.cbGet(gurobipy.GRB.Callback.MIPSOL_OBJBND),
-            solve_data.LB, solve_data.UB, round(get_main_elapsed_time(
-                solve_data.timing), 2)))
+    if solve_data.bound_improved:
+        update_gap(solve_data)
+    config.logger.info(solve_data.log_formatter.format(solve_data.mip_iter, 'LP', cb_opt.cbGet(gurobipy.GRB.Callback.MIPSOL_OBJ),
```

Comment:
I do not know if this corresponds to an LP, given that it comes from previous fixations of integer variables. How about restrLP (from restricted LP)?

Reply:
I think LP is enough here, since it's called LP/NLP-based branch and bound method.

Reply:
Yeah, the name of the algorithm is not very informative (n the name it is not clear that you are solving the LPs from a BB tree). Please change the name to `restrLP`